romeo_sensors_py/nodes/camera.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+import rospy
+from pepper_sensors.pepper_camera import PepperCam
```

Comment:
Pepper ? So what's the deal with romeo_camera.py ? Why is there a PepperCam in there too ?